lms/static/sass/base/_grid-settings.scss

```diff
@@ -1,16 +1,24 @@
-@import "neat/neat-helpers"; // or "neat-helpers" when in Rails
+@import "neat/neat-helpers";
 
-/* Change the grid settings */
+// Change the grid settings
 $max-width: 1200px;
-/* Override the default global box-sizing */
+
+// Override the default global box-sizing
 $border-box-sizing: false;
-/* Grid width variables */
-$large-min-width: 769px;
 
+// Grid width variables
+$large-min-width: 769px;
 
-/* Breakpoints */
+// breakpoints
 $bp-tiny: new-breakpoint(max-width 320px 4);
 $bp-small: new-breakpoint(min-width 321px max-width 540px, 4);
 $bp-medium: new-breakpoint(min-width 541px max-width 768px, 8);
 $bp-large: new-breakpoint(min-width $large-min-width max-width 979px, 12);
 $bp-huge:  new-breakpoint(min-width 980px 12);
+
+// edX-centric breakpoints (to be temporarily used with edx.org features only)
```

Comment:
@AlasdairSwan, here are your new breakpoints. Make sure I synced them up in all of the right places you folks need them for edx.org/DedX needs in the rest of the files, please.